src/sb_pointcloud_processing/include/DBSCAN.h

```diff
@@ -46,17 +50,25 @@ class DBSCAN {
      */
     vector<unsigned int>* _neighbors;
 
-    int _min_neighbors = 5;
-    float _radius      = 5;
-
+    int _min_neighbors               = 5;
+    float _radius                    = 5;
     unsigned int _sequential_cut_off = 1000;
```

Comment:
In general, we should avoid initializing member variables like this, and should instead use constructor initialization lists: https://www.cprogramming.com/tutorial/initialization-lists-c++.html